src/firebase/database.js

```diff
@@ -0,0 +1,2 @@
+// Firebase Init
+import { initializeApp } from "firebase/database";
```

Comment:
unused? Do we need this file?

Reply:
Yes, we're adding to it in other branches